packages/web-components/src/components/va-alert-expandable/va-alert-expandable.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Prop,
+  Listen,
+  State,
+  h,
+} from '@stencil/core';
+import classnames from 'classnames';
+
+@Component({
+  tag: 'va-alert-expandable',
+  styleUrl: 'va-alert-expandable.css',
+  shadow: true,
+})
+export class VaAlertExpandable {
+  @Element() el: HTMLElement;
+
+  @State() open: boolean;
+
+  /**
+   * Determines the icon and background color.
+   * One of `info`, `error`, `success`, `warning`, or `continue`
+   */
+  @Prop() status: string = 'info';
```

Comment:
This isn't crucial (and I know the `<va-alert>` doesn't do this yet), but instead of a plain `string` type can we define those options like this?

https://github.com/department-of-veterans-affairs/component-library/blob/6ee45b200d8be4a507b38dd1b2951d41f81e38c9/packages/web-components/src/components/va-modal/va-modal.tsx#L120